webapp/src/dogma/features/file/FileContentDto.ts

```diff
@@ -0,0 +1,7 @@
+export interface FileContentDto {
+  content: string;
+  name: string;
+  path: string;
+  revision: string;
+  type: 'TEXT' | 'DIRECTORY' | 'JSON' | 'YML';
```

Comment:
`YAML` type is going to be added.
https://github.com/line/centraldogma/pull/616/files#diff-aeb01a851ddc2caf419729e7ffbb8abbd7c86ed2371eaef7e17951b368d07ec5
```suggestion
  type: 'TEXT' | 'DIRECTORY' | 'JSON' | 'YAML';
```